binding-service-impl/src/test/java/io/mosip/esignet/KeyBindingServiceTest.java

```diff
@@ -119,6 +126,20 @@ public void sendBindingOtp_withInvalidRequest_thenFail() throws SendOtpException
 		keyBindingService.sendBindingOtp(otpRequest, headers);
 	}
 
+
+	@Test(expected = EsignetException.class)
+	public void sendBindingOtpV2_withCaptcha_thenFail() throws SendOtpException {
```

Comment:
Is this Junit test case a positive test or a negative test? If the captcha is valid, then sendBIndingOtp should pass, and it should fail only with an invalid captcha token.